tests/baselines/reference/jsdocLink1.baseline

```diff
@@ -0,0 +1,191 @@
+[
+  {
+    "marker": {
+      "fileName": "/tests/cases/fourslash/jsdocLink1.ts",
+      "position": 189,
+      "name": ""
+    },
+    "quickInfo": {
+      "kind": "function",
+      "kindModifiers": "",
+      "textSpan": {
+        "start": 189,
+        "length": 2
+      },
+      "displayParts": [
+        {
+          "text": "function",
+          "kind": "keyword"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "CC",
+          "kind": "functionName"
+        },
+        {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": ":",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "void",
+          "kind": "keyword"
+        }
+      ],
+      "documentation": [
+        {
+          "text": "",
+          "kind": "text"
+        },
+        {
+          "text": "\n",
+          "kind": "lineBreak"
+        },
+        {
+          "text": "{@link C}",
```

Comment:
Without having thought too deeply about it, I would have expected `C` to have its own display part.

Reply:
I thought about it, and it makes sense, as far as I can tell, from the perspective of an editor, to have 3 kinds of display parts for links: one for the surrounding `"{@link "`, one for the link name and one for the link text:

That is, `{@link C some class}` will be provided as 

* `"link": `{@link `
* `"linkName": `C` (plus a link to the span in the file where C is)
* `"linkText": ` some class`
* `"link"`: `}`

That should make it easy to hide parts of a link tag, plus associate the link target with its name and link text. @mjbvz does that make sense to you? The baselines in jsdocLink1,2,3 show this off.